### PR TITLE
Change activity to accept changes when active

### PR DESCRIPTION
**Pull request by Yelinz**

### CODE REVIEW

File: timed/tracking/serializers.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def validate(self, data):
 
         # validate that there is only one active activity
         activity = models.Activity.objects.filter(user=user)
-        if activity.filter(to_time__isnull=True) and to_time is None:
+        if (activity.filter(to_time__isnull=True) and
```
Comment (sliverc) [inline timed/tracking/serializers.py]: what happens when I patch activity and set `to_time` to `None`? I guess we could have more than one activity with `to_time` `None` right?

Comment (Yelinz) [reply timed/tracking/serializers.py]: I added an test and changed the condition

Comment (Yelinz) [reply timed/tracking/serializers.py]: Changed it to how we discussed it

File: timed/tracking/serializers.py [hunk 2]
```diff
@@ -39,12 +39,20 @@ def validate(self, data):
         to_time = data.get('to_time', instance and instance.to_time)
         user = instance and instance.user or data['user']
 
+        def validate_unique():
```
Comment (sliverc) [inline timed/tracking/serializers.py]: Sounds quite generic and could be anything. Maybe `validate_running_activity` might be a better name? Or any other ideas?

Comment (Yelinz) [reply timed/tracking/serializers.py]: Couldn't think of a better name, so i just used your suggestion

**Pull request merged**